trl/trainer/ppo_trainer.py

```diff
@@ -1336,17 +1337,16 @@ def log_stats(
                     raise ValueError(f"Columns to log {columns_to_log} are not present in the batch {batch.keys()}.")
 
                 batch_list = [batch[column_to_log] for column_to_log in columns_to_log]
+                if self.is_distributed:
+                    self.accelerator.wait_for_everyone()
+                    gathered_batch_list = []
+                    for b in batch_list:
```

Comment:
turbo nit to follow HF conventions to use descriptive names
```suggestion
                    for batch in batch_list:
```